internal/service/alarm_subscription_handler.go

```diff
@@ -271,7 +294,13 @@ func (h *alarmSubscriptionHandler) fetchItem(ctx context.Context,
 func (h *alarmSubscriptionHandler) fetchItems(ctx context.Context) (result data.Stream, err error) {
 	h.subscritionMapMemoryLock.Lock()
 	defer h.subscritionMapMemoryLock.Unlock()
-	ar := maps.Values(h.subscriptionMap)
+
+	ar := make([]data.Object, 0, len(h.subscriptionMap))
```

Comment:
To avoid duplicated subscription ID value in the configmap, I removed the subscription ID in data.object. This is the place we loop the datamap and encode the subscript ID back. later "stream" is used to build the http packet. Please let me know.